stats/kafka/collector.go

```diff
@@ -0,0 +1,125 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2016 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package kafka
+
+import (
+	"context"
+	"time"
+
+	"github.com/Shopify/sarama"
+	"github.com/loadimpact/k6/lib"
+	"github.com/loadimpact/k6/stats"
+	log "github.com/sirupsen/logrus"
+)
+
+// Collector implements the lib.Collector interface and should be used only for testing
```

Comment:
This looks like a copy-pasted comment :smile:. I really like how the kafka collector is shaping, so I sincerely hope that people use it a lot for actual work :grin:

Reply:
Did you want me to remove this and the below comment or update their text?

Reply:
Just update the text so it reflects the reality. The `dummy` collector from which this was copy-pasted is supposed to be used only for unit testing, that's why it says `should be used only for testing`